integration-test-groups/foundation/language/src/main/java/org/apache/camel/quarkus/component/language/it/LanguageResource.java

```diff
@@ -18,34 +18,27 @@
 
 import javax.enterprise.context.ApplicationScoped;
 import javax.inject.Inject;
-import javax.ws.rs.GET;
+import javax.ws.rs.Consumes;
+import javax.ws.rs.POST;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.MediaType;
-import javax.ws.rs.core.Response;
 
-import org.apache.camel.CamelContext;
-import org.jboss.logging.Logger;
+import org.apache.camel.ProducerTemplate;
 
 @Path("/language")
 @ApplicationScoped
 public class LanguageResource {
 
-    private static final Logger LOG = Logger.getLogger(LanguageResource.class);
-
-    private static final String COMPONENT_LANGUAGE = "language";
     @Inject
-    CamelContext context;
+    ProducerTemplate template;
 
-    @Path("/load/component/language")
-    @GET
+    @Path("/route/{route}")
+    @POST
+    @Consumes(MediaType.TEXT_PLAIN)
     @Produces(MediaType.TEXT_PLAIN)
-    public Response loadComponentLanguage() throws Exception {
-        /* This is an autogenerated test */
-        if (context.getComponent(COMPONENT_LANGUAGE) != null) {
-            return Response.ok().build();
-        }
-        LOG.warnf("Could not load [%s] from the Camel context", COMPONENT_LANGUAGE);
-        return Response.status(500, COMPONENT_LANGUAGE + " could not be loaded from the Camel context").build();
+    public String resource(String body, @PathParam("route") String route) {
```

Comment:
resource ? Maybe it could be renamed to something like invokeRoute ?

Reply:
Yep, good idea, renamed to `route`.